cmd/datasource-syncer/README.md

```diff
@@ -0,0 +1,63 @@
+# Datasource Syncer
+
+This CLI tool can act as a cron job remotely syncing the given Grafana Prometheus datasource, so it has e.g.:
+
+* Correct URL
+* Correct HTTP method
+* Fresh Oauth2 access token (valid for 1h)
+
+This allows you to directly query Google Cloud Monitoring with Grafana.
+
+[Google access tokens have a lifetime of 1 hour.](https://cloud.google.com/docs/authentication/token-types#at-lifetime) This script should be run hourly to ensure you have an uninterrupted connection between Grafana and Google Cloud Monitoring.
+
+### Run
+
+1. [Make sure you have setup your Grafana datasource setup.](https://cloud.google.com/stackdriver/docs/managed-prometheus/query#grafana-datasource) Extract datasource IDs you wish to use from URL.
```

Comment:
your Grafana Prometheus datasource set up. Extract the datasource IDs you wish to use from the URL.

Do we have an example datasource URL and ID?